_data/team.yml

```diff
@@ -33,9 +33,9 @@ en: &DEFAULT_EN
         social:
           - url: https://twitter.com/
             icon: fab fa-twitter
-          - url: https://github.com/
+          - url: https://github.com/dangerdika
             icon: fab fa-github
-          - url: https://www.linkedin.com/in/
+          - url: https://www.linkedin.com/
```

Comment:
I believe they were all originally meant to link to the homepage if we hadn't provided a URL to our profile, if any, but clicking on it with the original URL led to a 404 page. I'll raise an issue for everyone to update the file with their profile URLs.

Reply:
Ah ok, weird it worked for me. I would say don't worry about creating an issue. If people want to share their LinkedIn they can. The default should be that they logo is not there, rather than taking them to linkedin homepage. It would be bad experience for users because otherwise they'll think theyre going to someones profile page if the logo is there

Reply:
Sounds good, I've now removed the social media icons with no profile-specific URLs linked.